include/openssl/bio.h

```diff
@@ -447,6 +447,9 @@ OPENSSL_EXPORT int BIO_set_mem_buf(BIO *bio, BUF_MEM *b, int take_ownership);
 // default is -1 so that additional data can be written once exhausted.
 OPENSSL_EXPORT int BIO_set_mem_eof_return(BIO *bio, int eof_value);
 
+// Returns the normal BIO_METHOD |BIO_s_mem|. Deprecated since AWS-LC does not
+// support secure heaps.
+OPENSSL_EXPORT OPENSSL_DEPRECATED const BIO_METHOD *BIO_s_secmem(void);
```

Comment:
Maybe we should likewise mark `OPENSSL_secure_zalloc`, `OPENSSL_secure_malloc`, etc., as deprecated due to not supporting secure heaps?